src/containers/ForgotPassword/page.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+
+import Alert from 'components/Auth/Alert';
+import Logo from 'components/Auth/Logo';
+import BackgroundLayout from 'components/BackgroundLayout';
+import Form from 'containers/ForgotPassword/form';
+import backgroundImage from 'images/auth_background.png';
+
+const Page = () => {
+  const logoLabel =
+    'Enter your email to receive instructions for resetting your password.';
```

Comment:
minor thing but 1 newline here before `return` would be great.

Reply:
Added!